File: app/src/main/java/com/example/calcamp/view/MainActivity.java
package com.example.calcamp.view;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.View;

import com.example.calcamp.R;

public class MainActivity extends AppCompatActivity {

    private Object View;
    private Toolbar toolbar;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        CardView cardTeam = findViewById(R.id.cardTeam);
        CardView cardLeague = findViewById(R.id.cardLeague);

        cardTeam.setOnClickListener(v -> {
            Intent TeamActivity = new Intent(getApplicationContext(), com.example.calcamp.view.TeamActivity.class);
            startActivity(TeamActivity);
        });

        cardLeague.setOnClickListener(v ->{
                Intent LeagueActivity = new Intent(getApplicationContext(), com.example.calcamp.view.LeagueActivity.class);
                startActivity(LeagueActivity);
        });
    }
    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.menu_equipes, menu);
        return true;
    }

}

File: app/src/main/java/com/example/calcamp/view/kotlin/MainActivity.kt
package com.example.calcamp.view.kotlin

import android.R
import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.clickable
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.foundation.layout.fillMaxWidth
import androidx.compose.foundation.layout.padding
import androidx.compose.material3.Card
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.text.SpanStyle
import androidx.compose.ui.text.buildAnnotatedString
import androidx.compose.ui.text.withStyle
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.ui.unit.dp
import com.example.calcamp.view.kotlin.ui.theme.CalcampTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            CalcampTheme {
                // A surface container using the 'background' color from the theme
                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
                    Column(modifier = Modifier.padding(15.dp)) {
                        Greeting("Android")
                        CardDemo()
                    }
                }
            }
        }
    }
}
@Preview(showBackground = true)
@Composable
fun CardDemo(){
    Card(
        modifier = Modifier
            .fillMaxWidth()
            .padding(15.dp)
            .clickable { }
    ){
        Column(
            modifier = Modifier.padding(15.dp)
        ){
            Text(
                buildAnnotatedString {
                    append("welcome to")
                }
            )
            Text(
                buildAnnotatedString {
                    append("New app jetpack Compose")
                }
            )
        }
    }
}

@Composable
fun Greeting(name: String) {
    Text(text = "Hello $name!")
}

@Preview(showBackground = true)
@Composable
fun DefaultPreview() {
    CalcampTheme {
        Greeting("Android")
    }
}